.github/workflows/cast_vlaura_votes.yaml

```diff
@@ -0,0 +1,44 @@
+name: Post vlAURA snapshot votes to voter multisig and send to the vote relayer
+
+on:
+  workflow_dispatch:
+    inputs:
+      manual:
+        description: 'Use manual mode (true/false). Manual votes are placed in `tools/python/aura_snapshot_voting/manual_voting/{pool_type}.csv`'
+        required: true
+
+
+jobs:
+  generate_fees_report:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: vlAURA Voting
+      env:
+        ETHNODEURL: ${{ secrets.ETHNODEURL }}
+        PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+        DUNE_API_KEY: ${{ secrets.DUNE_API_KEY }}
+      run: |
+        pwd
+        cd tools/python
+        pip3 install -r requirements.txt
+        cd aura_snapshot_voting
+        python3 vote.py ${{ github.event.inputs.manual == 'true' && ' --manual' || '' }}
```

Comment:
I'm not familiar with this syntax, would have to see if it works.

Reply:
adjusted to be more intuitive, see: c73267f